sklearn/preprocessing/_encoders.py

```diff
@@ -215,7 +232,9 @@ class OneHotEncoder(_BaseEncoder):
         Specifies a methodology to use to drop one of the categories per
         feature. This is useful in situations where perfectly collinear
         features cause problems, such as when feeding the resulting data
-        into a neural network or an unregularized regression.
+        into a neural network or an unregularized regression. Drop is not
```

Comment:
instead of mentioning `min_frequency` and `max_categories`, would it be enough to mention `handle_unknown="infrequent_if_exist"`?

Reply:
I removed this sentence, since `drop` now works with infrequent categories.